packages/@romejs/diagnostics/descriptions.ts

```diff
@@ -402,6 +402,10 @@ export const descriptions = createMessages({
       category: 'lint/reactInJsxScope',
       message: markup`<emphasis>"React"</emphasis> must be in scope when using JSX`,
     },
+    REACT_JSX_A11Y_HTML_HAS_LANG: {
+      category: 'lint/jsxA11yHTMLHasLang',
+      message: markup`<emphasis>html</emphasis> elements must have a <emphasis>lang prop</emphasis>.`,
```

Comment:
`markup` is only necessary when interpolating values. I've noticed that usage of `markup` at the top level here will sometimes trigger module cycle errors in tests.

```suggestion
      message: `<emphasis>html</emphasis> elements must have a <emphasis>lang prop</emphasis>.`,
```